tests/scalability/json_generator.py

```diff
@@ -0,0 +1,273 @@
+import random
+import rstr
+from faker import Faker
+from jsonschema import RefResolver
+from typing import Union
+
+
+class JsonDataGenerator(object):
```

Comment:
JSON is always data. I would call it JsonGenerator

Reply:
fixed